Packs/CommonScripts/ReleaseNotes/1_19_15.md

```diff
@@ -0,0 +1,8 @@
+
+#### Scripts
+
+##### CommandLineAnalysis
+
+- Fix a bug in isBase64 function
+- Changed the output format of the extracted indicators
+- Added patterns to detect macOS Applescript suspicious command lines
```

Comment:
Should be:
```suggestion
- Fixed an issue in **CommandLineAnalysis** where <unexpected behaviour>.
- Updated the output format of the extracted indicators.
- Added the patterns to detect macOS Applescript suspicious command lines.
```
See https://xsoar.pan.dev/docs/documentation/release-notes#format for more info.